.github/workflows/unit-tests.yml

```diff
@@ -51,7 +51,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [2.7, 3.5, 3.8]
+        python-version: [3.6, 3.8, 3.10.0]
```

Comment:
Hijacking the comment on this line, is there any reason the hardcode the .0 minor version in python 3.10. Ideally we want to test if it works with new stable releases as they come instead of pinning to a version.